src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -181,4 +185,40 @@ public static HttpHost getBriefCaseProxyConnection() {
     }
     return null;
   }
+
+  /**
+   * Persist the user's decision to allow/disallow their behaviour being tracked.
+   * @param value (required) the boolean value representing the user's decision.
+   */
+  public static void setBriefcaseTrackingConsentProperty(Boolean value) {
+    if (value == null) {
+      Preference.APPLICATION_SCOPED.remove(BRIEFCASE_TRACKING_CONSENT_PROPERTY);
+    } else {
+      Preference.APPLICATION_SCOPED.put(BriefcasePreferences.BRIEFCASE_TRACKING_CONSENT_PROPERTY, value.toString());
+    }
+  }
+
+  /**
+   * Get the user's persisted decision regarding their consent to being tracked.
+   * @return the boolean representation of the user's consent to being tracked.
+   */
+  public static Boolean getBriefcaseTrackingConsentProperty() {
```

Comment:
I was against doing this until I found both JCheckedBox.isSelected() and GoogleAnalyticsConfig.setGatherStats() use boolean instead of Boolean. My choice to go with Boolean was because it made persisting it easier because of how it _has_ to be stored as a String, ergo Boolean.toString(). I've read that it's bad to jump from similar type to similar type (Boolean to boolean and vice versa), which I couldn't avoid even though I thought I was doing so. Upon learning about the two previously mentioned signatures I decided it's probably best to keep the complexity of storing it inside the class responsible for storing it, and to keep using boolean everywhere else.